src/lib/extensions/dashboard/DashboardExtension.ts

```diff
@@ -11,9 +13,14 @@ export class DashboardExtension<ParamType extends Params = Params> extends Exten
    */
   public params!: ParamType;
   /**
+   * Deprecated
```

Comment:
maybe state what it has been deprecated for

Reply:
> Deprecated - has been superseded by the `hub` object.